doc/source/examples/extended_examples/sfem/stochastic_fem.rst

```diff
@@ -0,0 +1,434 @@
+over a time interval :math:`I \subseteq \mathbb{R}^+`. In such cases, the system's response at a specific material point is
+described not by a single random variable but by a collection of random variables :math:`\{X(t)\}` indexed by :math:`t \in I`. 
+
+This 'infinite' collection of random variables over the interval :math:`I` is called a stochastic process and is denoted as
+:math:`\{X(t), t \in I\}` or simply :math:`X`. In this way, a stochastic process generalizes the concept of a random variable
+as it assigns to each outcome :math:`\theta` of the experiment a function :math:`X(t, \theta)`, known as a realization or sample
+function. Lastly, if :math:`X` is indexed by some spatial coordinate :math:`s \in D \subseteq \mathbb{R}^n` rather than time :math:`t`,
+then :math:`\{X(s), s \in D\}` is called a random field.
+
+Practical example
++++++++++++++++++
+Consider the material property of the beam, such as Young's modulus :math:`E(x)`, which may vary randomly along
+the length of the beam :math:`x`.  Instead of being a single random value, :math:`E(x)` is a random field. Its value
+is uncertain at each point along the domain, and it changes continuously across the beam. Mathematically, :math:`E(x)`
+is a random field:
+
+.. math:: E(x) : x \in [0,L] \longrightarrow \mathbb{R}
+
+Here:
+
+* :math:`x` is the spatial coordinate along the length of the beam (:math:`x \in [0,L]`).
+* :math:`E(x)` is a random variable at each point :math:`x`, and its randomness is described
+  by a covariance function or an autocorrelation function.
+
+For example, :math:`E(x)` could be a Gaussian random field, in which case it has the stationarity
+property, making its statistics completely defined by its mean (:math:`\mu_E`), standard deviation
+(:math:`\sigma_E`) and covariance function :math:`C_E(x_i,x_j)`. This 'stationarity' simply means
+that the mean and standard deviation of every random variable :math:`E(x)` is constant and equal to
+:math:`\mu_E` and :math:`\sigma_E` respectively. :math:`C_E(x_i,x_j)` describes how random variables
+:math:`E(x_i)` and :math:`E(x_j)` are related. For a zero-mean Gaussian random field, the covariance function is given by
+this equation:
+
+.. math:: C_E(x_i,x_j) = \sigma_E^2e^{-\frac{\lvert x_i-x_j \rvert}{\ell}}
+
+Here :math:`\sigma_E^2` is the variance, and :math:`\ell` is the correlation length parameter.
+
+To aid understanding, the figure below is a diagram depicting two equivalent ways of visualizing a
+stochastic process or random field, that is, as an infinite collection of random variables or as a
+realization or sample function assigned to each outcome of an experiment.
+
+.. figure:: realizations.png
+
+   A random field :math:`E(x)` viewed as a collection of random variables or as realizations.
+
+.. note::
+  The concepts in the preceding sections generalize to more dimensions, for example, a random vector instead of a random
```

Comment:
```suggestion
  The concepts in the preceding topics generalize to more dimensions, for example, a random vector instead of a random
```
I try to save use of the word "section" to refer to the list appearing in the doc in the left navigation pane (labeled "Section Navigation").  They entries in the right navigation pane (labeled "On this page") are then referred to as topics.